app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,9 @@
 class PagesController < ApplicationController
   def home
   end
+
+  def message
+    render(json: { message: "Hello World" }, status: :unprocessable_entity)
```

Comment:
Maybe change the status to one that represents the request being ok? as this isn't and  error.